.github/workflows/test.yml

```diff
@@ -1,5 +1,9 @@
 name: Tests
-on: [pull_request, push]
```

Comment:
@cclauss I removed push because it was causing duplicate builds for branches that didn't come from a fork, see #82

Reply:
I believe that the way this works is...  If someone who is a repo maintainer (has write permissions to the repo) pushes a change to master then no test is run.  If that is true and that is an acceptable risk then go for it.

Reply:
Yeah pretty much the plan is PR only unless its a docs thing. Could always change it in the future or add a "manual" button to run from the actions page.